### PR TITLE
Fix edit page

### PR DESCRIPTION
**Pull request by Drewbi**

### CODE REVIEW

File: client/src/components/ProfileCard/index.js [hunk 1]
```diff
@@ -2,49 +2,39 @@ import React from "react";
 import { Link } from "react-router-dom";
 import classes from "./index.module.css";
 import defaultPicture from "../../assets/user_icon.png";
-import { makeStyles } from "@material-ui/core/styles";
 import { Button } from "@material-ui/core";
 
-const useStyles = makeStyles((theme) => ({
```
Comment (Drewbi) [inline client/src/components/ProfileCard/index.js:8]: This wasn't being used by anything

File: client/src/components/ProfileCard/index.js [hunk 2]
```diff
@@ -2,49 +2,39 @@ import React from "react";
...
     <div className={classes.card}>
       <div>
         <div className={classes.intro}>
-          <div className={classes.profileimage}>
-            <img
-              className={classes.bannerphoto}
-              src={data.picture ? data.picture : defaultPicture}
-              alt={data.name}
-            ></img>
+          <div>
```
Comment (Drewbi) [inline client/src/components/ProfileCard/index.js:13]: Made spacing changes so buttons are always at bottom

File: client/src/components/ProfileCard/index.js [hunk 3]
```diff
@@ -2,49 +2,39 @@ import React from "react";
...
+                className={
+                  props.expert
+                    ? classes.bannername
+                    : `${classes.bannername} ${classes.banner_margin_bottom}`
+                }
+              >
+                {data.name}
+              </p>
+              <p className={classes.endorsment}>{`Endorsed by ${
+                Math.floor(Math.random() * 3) === 0
```
Comment (Drewbi) [inline client/src/components/ProfileCard/index.js:32]: Added endorsement randomiser

File: client/src/components/ProfileCard/index.module.css [hunk 4]
```diff
@@ -15,7 +15,7 @@
 }
 
 .card {
-  background-color: #1d1d1d;
+  background-color: #404040;
```
Comment (Drewbi) [inline client/src/components/ProfileCard/index.module.css:18]: Changed card color to be correct shade

File: client/src/components/Register/index.js [hunk 5]
```diff
@@ -34,7 +34,7 @@ const Register = ({ onClose, open }) => {
       }
     };
     getClaims();
-  }, []);
+  }, [getIdTokenClaims]);
```
Comment (Drewbi) [inline client/src/components/Register/index.js:37]: Fixing error

File: client/src/pages/Experts/index.module.css [hunk 6]
```diff
@@ -27,14 +27,6 @@
   font-size: 30px;
 }
 
-.card {
```
Comment (Drewbi) [inline client/src/pages/Experts/index.module.css:30]: Not used

**Review by khattakdev — APPROVED** [review]

**Pull request merged**